src/Jank/TermManagement/ui.tsx

```diff
@@ -0,0 +1,34 @@
+import styled from 'styled-components';
+
+export const Button = styled.button`
```

Comment:
As this is just used internally, you could consider not exporting it. This is similar to having a `private` member in other programming languages, which is usually a good idea, unless you need it to be `public`.
```suggestion
const Button = styled.button`
```